crypto/aes/asm/bsaes-armv8.pl

```diff
@@ -1029,7 +1029,7 @@ sub data
 
         // it is up to the caller to make sure we are called with enc == 0
 
-        stp     fp, lr, [sp, #-48]!
+        stp     x29, x30, [sp, #-48]!
```

Comment:
Maybe a comment here saying why the x29, x30 aliases of fp,lr are not used would be good.

Reply:
`fp` is rarely used as an alias; `lr` is meant to be a synonym for `x30`, so this is clearly a bug in old gcc, but this information is already captured in the commit message. I don't feel putting such a comment in one place in the code would be much better than having it in the commit message, and scattering such a comment everywhere `x29` or `x30` are used would be a distraction from the code.